<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetForm</class>
 <widget class="QWidget" name="BudgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../PiTres.qrc">
    <normaloff>:/icons/resources/icons/chart.png</normaloff>:/icons/resources/icons/chart.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolBar" name="horizontalWidget">
         <addaction name="actionCreateRoot"/>
         <addaction name="actionCreateFolder"/>
         <addaction name="actionRemoveFolder"/>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolBar" name="horizontalWidget_2">
         <addaction name="actionCreateEntry"/>
         <addaction name="actionRemoveEntry"/>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;b&gt;Total:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTotal">
           <property name="text">
            <string>0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="detailArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="detailLayout">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>131</width>
            <height>477</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;b&gt;Summary&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Budget:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelSummaryBudget">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Position:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelSummaryPosition">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Budgeted:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelSummaryBudgeted">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Spent:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="labelSummarySpent">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>&lt;b&gt;Total:&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="labelSummaryTotal">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>&lt;b&gt;Position&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Budgeted:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="labelOverviewBudgeted">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Spent:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="labelOverviewSpent">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>&lt;b&gt;Total:&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="labelOverviewTotal">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="labelOverviewPosition">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionCreateFolder">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/folders.png</normaloff>:/icons/resources/icons/folders.png</iconset>
   </property>
   <property name="text">
    <string>Create Position</string>
   </property>
   <property name="toolTip">
    <string>Create a new Position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemoveFolder">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/folder_delete.png</normaloff>:/icons/resources/icons/folder_delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Position</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected Position and all entries and Subpositions</string>
   </property>
  </action>
  <action name="actionCreateEntry">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/notebook.png</normaloff>:/icons/resources/icons/notebook.png</iconset>
   </property>
   <property name="text">
    <string>Create Entry</string>
   </property>
   <property name="toolTip">
    <string>Create a new Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemoveEntry">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/delete.png</normaloff>:/icons/resources/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Entry</string>
   </property>
   <property name="toolTip">
    <string>Remove the Entry</string>
   </property>
  </action>
  <action name="actionCreateRoot">
   <property name="icon">
    <iconset resource="../PiTres.qrc">
     <normaloff>:/icons/resources/icons/chart.png</normaloff>:/icons/resources/icons/chart.png</iconset>
   </property>
   <property name="text">
    <string>New Budget</string>
   </property>
   <property name="toolTip">
    <string>Create a new Budget</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../PiTres.qrc"/>
 </resources>
 <connections/>
</ui>
